beancount/loader_test.py

```diff
@@ -704,7 +733,8 @@ def test_load_cache_disable(self):
                     },
                 )
                 filename = path.join(tmp, "apples.beancount")
-                entries, errors, options_map = loader.load_file(filename)
+                with mock_time_time():
```

Comment:
This test function is executed with `PICKLE_CACHE_THRESHOLD` set to 0. Thus this should not be necessary.